lantern/django/django_orm/src/config/settings/local.py

```diff
@@ -1,4 +1,4 @@
-from .base import *  #noqa
+from .base import *
```

Comment:
please avoid `import *` better would be to specify all imported variables, functions and classes